src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,188 @@
+
+  if (!sortedPeople.length) {
+    return <p>There are no people matching the current search criteria</p>;
+  }
+
+  const updateSearchParams = (sortBy: SortBy) => {
+    if (!sort || sort !== sortBy) {
+      return ({
+        sort: sortBy,
+        order: null,
+      });
+    }
+
+    if (sort && !isReversed) {
+      return ({
+        sort: sortBy,
+        order: 'desc',
+      });
+    }
+
+    return ({
+      sort: null,
+      order: null,
+    });
+  };
+
+  const handleSortStyle = (sortBy: SortBy) => {
+    if (sort === sortBy && !isReversed) {
+      return 'fas fa-sort-up';
+    }
+
+    if (sort === sortBy && isReversed) {
+      return 'fas fa-sort-down';
+    }
+
+    return 'fas fa-sort';
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Name
+              <SearchLink
+                params={updateSearchParams(SortBy.Name)}
+              >
+                <span className="icon">
+                  <i className={handleSortStyle(SortBy.Name)} />
+                </span>
+              </SearchLink>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Sex
+              <SearchLink
+                params={updateSearchParams(SortBy.Sex)}
+              >
+                <span className="icon">
+                  <i className={handleSortStyle(SortBy.Sex)} />
+                </span>
+              </SearchLink>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Born
+              <SearchLink
+                params={updateSearchParams(SortBy.Born)}
+              >
+                <span className="icon">
+                  <i className={handleSortStyle(SortBy.Born)} />
+                </span>
+              </SearchLink>
+            </span>
+          </th>
+
+          <th>
+            <span className="is-flex is-flex-wrap-nowrap">
+              Died
+              <SearchLink
+                params={updateSearchParams(SortBy.Died)}
+              >
+                <span className="icon">
+                  <i className={handleSortStyle(SortBy.Died)} />
+                </span>
+              </SearchLink>
+            </span>
+          </th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {sortedPeople.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+            slug,
+          } = person;
+
+          const hasSelected = slug === personSlug;
+          const selectedMother = people.find(mom => mom.name === motherName);
+          const selectedFather = people.find(dad => dad.name === fatherName);
```

Comment:
it's better to prepare people's data before rendering because now you use the find method each time when the component is rendered. but you can do it once when data loaded from the server. So you can moove this logic to the halper.